### PR TITLE
Refactor rlrop condition

### PR DESCRIPTION
**Pull request by DanBmh**
Improve readability.
Split to separate pr from #3268.

### CODE REVIEW

File: training/deepspeech_training/train.py [hunk 1]
```diff
@@ -640,8 +640,8 @@ def __call__(self, progress, data, **kwargs):
                         break
 
                     # Reduce learning rate on plateau
-                    if (FLAGS.reduce_lr_on_plateau and
-                            epochs_without_improvement % FLAGS.plateau_epochs == 0 and epochs_without_improvement > 0):
+                    if (FLAGS.reduce_lr_on_plateau and epochs_without_improvement > 0
```
Comment (lissyx) [inline training/deepspeech_training/train.py:643]: let's improve a bit more and put each condition on a new line, ending in `and` ?

**Review by lissyx — CHANGES_REQUESTED** [review]
LGTM but let's take this to the maximum :)

**Comment by DanBmh** [comment]
Now it's following black style

**Review by lissyx — CHANGES_REQUESTED** [review]
It's good but your commit is mixed with unrelated things, please rebase properly on master :/

**Comment by DanBmh** [comment]
> It's good but your commit is mixed with unrelated things, please rebase properly on master :/

?
Did you mix this up with another PR?

**Comment by DanBmh** [comment]
I can see now what you mean (clicking on your comment links) but I didn't change those files. 

Your link says: Showing 3 changed files with 10 additions and 8 deletions. 
But I can only see 1 changed file with 5 additions and 3 deletions in the overview here

**Comment by reuben** [comment]
It looks like you merged into this PR instead of rebasing it.

**Comment by reuben** [comment]
At the bottom of the changes page GitHub shows: You are viewing a condensed version of this merge commit. You can view the full changes here.

**Comment by DanBmh** [comment]
Thanks @reuben. Now it should be right.

**Review by lissyx — APPROVED** [review]

**Comment by lissyx** [comment]
> Thanks @reuben. Now it should be right.

Thanks!

**Pull request merged**